tiatoolbox/dataloader/wsireader.py

```diff
@@ -84,15 +85,148 @@ def slide_info(self):
         """
         raise NotImplementedError
 
-    def read_region(self, start_w, start_h, end_w, end_h, level=0):
+    def relative_level_scales(self, target_scale, units):
+        """
+        Calculate scale of each image pyramid level relative to the
+        given target scale and units.
+
+        Values > 1 indicate that the level has a larger scale than the
+        target and < 1 indicates that it is smaller.
+
+        Args:
+            target_scale (float): Scale to calculate relative to
+            units (str): Units of the scale. Allowed values are: mpp,
```

Comment:
What is the difference between the units? What is the expected output for each? It's not obvious to me what will be the output when using `units='level'` here. I think you neeed to provide examples in docstr for all of them.

Reply:
There are quite a few examples in the docs now. Have a look at https://tia-toolbox.readthedocs.io/en/feature-calc-scale/. 

I think we should look at adding a "getting started" page to these docs too. Similar to Glymur or Pillow.